src/core/string_set_test.cc

```diff
@@ -135,27 +135,30 @@ static string random_string(mt19937& rand, unsigned len) {
 
 TEST_F(StringSetTest, Resizing) {
   constexpr size_t num_strs = 4096;
-  vector<string> strs;
+  unordered_set<string> strs;
```

Comment:
why?

Reply:
If you want to make it unique, std::unique is a better option

Reply:
it's not better. it assumes the vector is sorted. this is simpler.